css/home.css

```
@@ -0,0 +1,215 @@
+}
+
+.hero-content__title {
+  font-weight: 700;
+  font-size: 40px;
+}
+
+.hero__image {
+  width: 746px;
+  height: 340px;
+}
+
+.btn-items {
+  width: 357px;
+  height: 56px;
+  padding: 16px 124px 16px 124px;
+  border-radius: 40px;
+  color: var(--secondary-gray-50);
+  background-color: var(--primary-100);
+  font-size: 20px;
+  font-weight: 600;
+}
+
+.hot-item,
+.register {
+  background-color: #fff;
+  display: flex;
+  justify-content: center;
+}
+
+.hot-item-container,
+.register-container {
+  display: flex;
+  align-items: center;
+  padding-right: 40px;
+  margin: 100px 0 100px -150px;
+  gap: 64px;
+  background: #fcfcfc;
+}
+
+.hot-item-content,
+.register-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.hot-item-content__span,
+.register-content__span {
+  color: var(--primary-100, #3692ff);
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 26px;
+}
+
+.hot-item-content__description,
+.register-content__description {
+  color: var(--secondary-gray-700, #374151);
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 32px;
+}
+
+.hot-item-content__title,
+.register-content__title {
+  color: var(--secondary-gray-700, #374151);
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 56px;
+  letter-spacing: 0.8px;
+  margin-top: 12px;
+  margin-bottom: 24px;
+}
+
+.search {
+  background-color: #fff;
+  display: flex;
+  justify-content: center;
+}
+
+.search-container {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  padding-left: 40px;
+  margin: 100px -150px 100px 0;
+  gap: 64px;
+  background: #fcfcfc;
+}
+
+.search-content {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  text-align: end;
+  gap: 24px;
+}
+
+.search-content__span {
+  color: var(--primary-100, #3692ff);
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 26px;
+}
+
+.search-content__description {
+  color: var(--secondary-gray-700, #374151);
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 32px;
+}
+
+.search-content__title {
+  color: var(--secondary-gray-700, #374151);
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 56px;
+  letter-spacing: 0.8px;
+}
+
+.bottom {
+  background-color: var(--main-bg-color);
+  display: flex;
+  justify-content: center;
+}
+
+.bottom-container {
+  height: 540px;
+  display: flex;
+  align-items: flex-end;
+  gap: 69px;
+}
+
+.bottom-content {
+  margin-bottom: 170px;
+}
+
+.bottom-content__title {
+  color: var(--secondary-gray-700, #374151);
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 56px;
+}
+
+.footer-container {
+  height: 160px;
+  display: flex;
+  justify-content: space-between;
+  padding-top: 32px;
+}
+
+.copyright {
+  color: #9ca3af;
+}
+
+.footer-mid {
+  display: flex;
+  justify-content: space-between;
+  gap: 30px;
+}
+
+.footer-mid__a {
```

P3:
태그명은 클래스명에 들어가지 않는 것을 추천드립니다.
좀더 의미가 있는 클래스명을 추천드립니다.

```suggestion
.footer-mid__link {
```